e2e_tests/test_browse.py

```diff
@@ -0,0 +1,92 @@
+import time
+
+from playwright.sync_api import Page
+
+
+def test_has_title(page: Page):
+    time.sleep(1)
+    page.goto("/browse")
+    assert page.title() == "Browse – AYR - Access Your Records – GOV.UK"
+
+
+def test_sort_dropdown(page: Page):
+    time.sleep(1)
+    page.goto("/browse")
+    page.select_option("#sort", "body-a")
+    page.select_option("#sort", "body-b")
+    page.select_option("#sort", "series-a")
+    page.select_option("#sort", "series-b")
+    page.select_option("#sort", "date-first")
+    page.select_option("#sort", "date-last")
+
+
+def test_page_loading(page: Page):
+    time.sleep(1)
+    page.goto("/browse")
+    page.wait_for_selector(".govuk-heading-l")
+
+
+def test_filter_functionality(page: Page):
+    page.goto("/browse")
+
+    page.select_option(
+        ".filters-form__transferring-body--select", "Arts Council England"
+    )
+    time.sleep(1)
+
+    records = page.query_selector_all(".govuk-summary-list__value a")
+    for record in records:
+        assert record
+        # assert record.text() == "Arts Council England"
+
+
```

Comment:
I guess this PR isnt ready to review yet until Pavans work is merged in?

Reply:
Yes, this will be updated with the ui revisions & test updates to confirm actual records that should be visible to our test users.